### ISSUE TITLE
NCHW格式的节点和NC4HW4格式的节点在经过torch.concat（）之后的输出全变为0

### ISSUE THREAD

**Issue by lzylzy2223** [alibaba/MNN#1513]
audio_out = self.audio_output(input_audio)
        #
        audio_out = self.audio_tcn(audio_out)
        # audio_out will be (N,256,298,1)
        AVFusion = [audio_out]
        # # for i in range(self.num_person):
        # #     video_out = self.video_output(input_video[i])
        video_out = self.video_output(input_video)
        AVFusion.append(video_out)
        # # print (len(AVFusion)
        mixed_av = torch.cat(AVFusion, dim=1)

mnn模型输出的时候video_out和audio_out都是正常的，mixed_av输出后全变成0，其中input_audio是NC4HW4格式，input_video是NCHW格式

**Comment by jxt1234** [comment]
先参考 https://www.yuque.com/mnn/cn/model_convert 里面用 onnx 的正确性校验对一下结果
